codebuild/spec/buildspec_ktls.yml

```diff
@@ -8,17 +10,27 @@ phases:
       - apt install -y qemu qemu-system-x86 wget cloud-image-utils
   pre_build:
     commands:
+      - mkdir ~/.ssh; echo -e "Host localhost\nUser codebuild\nPort 2222" > ~/.ssh/config
       - aws s3 --quiet sync s3://s2n-ktls-testing ./qemu
       - cd qemu; bash ./run.sh; cd ..
       - rsync -avz --exclude=qemu --exclude=tests/fuzz -e 'ssh -p 2222' . codebuild@localhost:/home/codebuild/s2n-tls
   build:
     commands:
-      - codebuild-breakpoint
       - |
-        ssh -p 2222 codebuild@localhost " \
+        ssh localhost " \
             cd s2n-tls; sudo modprobe tls; \
             export S2N_CMAKE_OPTIONS=${S2N_CMAKE_OPTIONS}; \
             export S2N_KTLS_TESTING_EXPECTED=1; \
-            nix develop .#openssl111 --command bash -c \
-                'source ./nix/shell.sh && clean && configure && unit' \
+            nix develop .#ktls --command bash -c \
+                'source ./nix/shell.sh && clean && configure' \
+        "
+  post_build:
+    commands:
```

Comment:
Why is this post build? Do we run the unit tests as post_build instead of build in other places? It also seems simpler to leave it as build.

Reply:
I've updated the names to make more sense; the end goal is to track timing of the build vs the test run

Reply:
Oh splitting them to track timing sounds reasonable.

In that case, I strongly prefer build + post_build vs pre_build vs build. It seems really weird to be building in pre_build. Technically unit tests are post_build, we just usually treat them as part of the build.